src/sigma.cpp

```diff
@@ -185,13 +185,21 @@ bool CheckSigmaSpendTransaction(
         uint256 hashTx,
         bool isVerifyDB,
         int nHeight,
+        int nRealHeight,
         bool isCheckWallet,
         bool fStatefulSigmaCheck,
         CSigmaTxInfo *sigmaTxInfo) {
     bool hasSigmaSpendInputs = false, hasNonSigmaInputs = false;
     int vinIndex = -1;
     std::unordered_set<Scalar, sigma::CScalarHash> txSerials;
 
+    Consensus::Params const & params = ::Params().GetConsensus();
+
+    if(!isVerifyDB && !isCheckWallet) {
```

Comment:
Just noticed that `isVerifyDB` is always passed as false to this function. Is it necessary to keep it?

Reply:
it is code cleanup, better do it with another PR I think.

Reply:
It was my initial implementation but I was asked to make the change smaller.